fabfile.py

```diff
@@ -93,27 +148,127 @@ def listdir(path):
 
 
 @task
-def database_start():
-    """
-    Makes sure the database server is running
-    """
-    sanitize_env()
-    if using_virtuoso():
-        execute(supervisor_process_start, 'virtuoso')
+def create_local_ini():
+    """Replace the local.ini file with one composed from the current .rc file"""
+    random_ini_path = os.path.join(env.projectpath, env.random_file)
+    local_ini_path = os.path.join(env.projectpath, 'local.ini')
+    if exists(local_ini_path):
+        run('cp %s %s.%d' % (
+            local_ini_path, local_ini_path, int(time())))
+
+    if env.host_string == 'localhost':
+        # The easy case
+        venvcmd("python assembl/scripts/ini_files.py compose -o local.ini " + env.rcfile)
+    else:
+        # get placeholder filenames
+        with NamedTemporaryFile(delete=False) as f:
+            random_file_name = f.name
+        with NamedTemporaryFile(delete=False) as f:
+            local_file_name = f.name
+        try:
+            # remote server case
+            # Load the random file if any in a temp file
+            if exists(random_ini_path):
+                get(random_ini_path, random_file_name)
+            rt = os.path.getmtime(random_file_name)
+            # create the local.ini in a temp file
+            venvcmd("python assembl/scripts/ini_files.py compose -o %s -r %s %s" % (
+                local_file_name, random_file_name, env.rcfile))
+            # send the random file if changed
+            if rt != os.path.getmtime(random_file_name):
+                put(random_file_name, random_ini_path)
+            # send the local file
+            put(local_file_name, local_ini_path)
+        finally:
+            os.unlink(random_file_name)
```

Comment:
AH, `unlink` is same as remove. You're just deleting a file. Nothing to see here then...